### PR TITLE
Implement feature application-and-offers binding

### PR DESCRIPTION
**Pull request by alanhanaev**

### CODE REVIEW

File: src/content/lib/ApplicationsAndOffersContent/ApplicationsAndOffersContent.js [hunk 1]
```diff
@@ -108,54 +107,26 @@ class ApplicationsAndOffersContent extends React.Component {
...
-          },
-        },
-      ],
-    }
-  } else {
+
+  const myOffers=offersListSelector()(state)
+
+  const applicationsApproved = myOffers.map((offer) => {
+    const job=jobByIdSelector(offer.jobId)(state)
```
Comment (sargonpiraev) [inline src/content/lib/ApplicationsAndOffersContent/ApplicationsAndOffersContent.js:114]: spaces between = sign

File: src/store/offers/actions.js [hunk 2]
```diff
@@ -1,17 +1,30 @@
...
+import { JOB_STATE_CREATED } from "./../../models/app/JobStateModel"
 
 export const OFFERS_CLEAR = 'offers/clear'
 export const OFFERS_SAVE = 'offers/save'
 
 // Should be called only once
-export const initJobOffers = () => async (/*dispatch, getState*/) => {
-  // const state = getState()
-  // TODO aevalyakin subscribe on events here
+export const initJobOffers = () => async ( dispatch) => {
```
Comment (sargonpiraev) [inline src/store/offers/actions.js:9]: remove space before dispatch

File: src/store/offers/actions.js [hunk 3]
```diff
@@ -1,17 +1,30 @@
...
-export const reloadJobsOffers = ({ jobId }) => async (dispatch, getState) => {
+export const reloadJobsOffers = () => async (dispatch, getState) => {
   const state = getState()
   const jobDataProviderDAO = daoByType('JobsDataProvider')(state)
-  const offers = await jobDataProviderDAO.getJobOffers(jobId)
-  dispatch({ type: OFFERS_SAVE, jobId, offers })
+  const signer = signerSelector()(state)
+  if (state.jobs && state.jobs.list) {
+    for (let job of state.jobs.list) {
+      if (job.state===JOB_STATE_CREATED)
```
Comment (sargonpiraev) [inline src/store/offers/actions.js:19]: spaces around ===

File: src/store/offers/actions.js [hunk 4]
```diff
@@ -1,17 +1,30 @@
...
-  const offers = await jobDataProviderDAO.getJobOffers(jobId)
-  dispatch({ type: OFFERS_SAVE, jobId, offers })
+  const signer = signerSelector()(state)
+  if (state.jobs && state.jobs.list) {
+    for (let job of state.jobs.list) {
+      if (job.state===JOB_STATE_CREATED) 
+      {setTimeout(async () => {
+        const allOffers = await jobDataProviderDAO.getJobOffers(job.id)
+        //Filtering only my offers
+        const offers=allOffers.filter((item) => item.worker===signer.address )
```
Comment (sargonpiraev) [inline src/store/offers/actions.js:23]: spaces around = and ===

File: src/store/offers/actions.js [hunk 5]
```diff
@@ -1,17 +1,30 @@
...
+        if (offers.length>0)
```
Comment (sargonpiraev) [inline src/store/offers/actions.js:24]: spaces around >

File: src/store/offers/actions.js [hunk 6]
```diff
@@ -1,17 +1,30 @@
...
+        {dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })}
```
Comment (sargonpiraev) [inline src/store/offers/actions.js:25]: if (offers.length>0) {
--dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })
}

not 
if (offers.length>0)
{dispatch({ type: OFFERS_SAVE, jobId: job.id, offers  })}

**Review by sargonpiraev — CHANGES_REQUESTED** [review]

**Comment by sargonpiraev** [comment]
create pull request from feature/*** branch not from develop

**Comment by valyakin** [comment]
We use yarn not npm to manage dependencies. No need in package-lock.json

**Pull request closed without merging**